docs/reference-manual/native-image/JCmd.md

```diff
@@ -0,0 +1,83 @@
+| GC.heap_dump         | heapdump                             | Generate a HPROF format dump of the Java heap.                                                     |
+| GC.run               | Always available                     | Call `java.lang.System.gc()`.                                                                        |
+| JFR.start            | jfr                                  | Starts a new JFR recording.                                                                        |
+| JFR.stop             | jfr                                  | Stops a JFR recording.                                                                             |
+| JFR.check            | jfr                                  | Checks running JFR recording(s)                                                                    |
+| JFR.dump             | jfr                                  | Copies contents of a JFR recording to file. Either the name or the recording id must be specified. |
+| Thread.dump_to_file  | Always available                     | Dump threads, with stack traces, to a file in plain text or JSON format.                           |
+| Thread.print         | Always available                     | Print all threads with stacktraces.                                                                |
+| VM.command_line      | Always available                     | Print the command line used to start this VM instance.                                             |
+| VM.native_memory     | nmt                                  | Print native memory usage.                                                                         |
+| VM.system_properties | Always available                     | Print native memory usage                                                                          |
+| VM.uptime            | Always available                     | Print VM uptime.                                                                                   |
+| VM.version           | Always available                     | Print JVM version information.                                                                     | 
+| help                 | Always available                     | Display help information.                                                                          |
+
+## Performance
+
+The adding `jcmd` support will have minimal impact on Native Image performance while idle.  However, the performance impact of the `jcmd` feature greatly depends on which diagnostic commands are invoked and their frequency. For example, invoking multiple garbage collections will have a much greater impact than dumping a single native memory tracking report. 
+
+
+## Limitations
+
+Currently, this feature is not available on Windows.
+
+### Further Reading
+
+- [Debugging and Diagnostics](DebuggingAndDiagnostics.md)
```

Comment:
I would also add some JFR guide to this reading suggestion:
```suggestion
- [Build and Run Native Executables with JFR](guides/build-and-run-native-executable-with-jfr.md)
```

Reply:
Good idea! Applied!